README.md

```diff
@@ -210,11 +228,56 @@ public String getFoo() {
     // This error will not cause the span to have error tag
     throw new InvalidArgumentException();
 }
+```
+
+### Track span context across threads
+
+Use the Java Agent to track span context across thread boundaries. Provide
```

Comment:
"boundaries, assuming your asynchronous or concurrent workers are supported."  If not this has no effect.